ci/delete_thin-edge/delete_thin_edge.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -euo pipefail
+
+usage() {
+    cat <<EOF
+USAGE:
+    delete_thin_edge [remove/purge]
+EOF
+}
+
+stop_a_service_if_running() {
+    local status
+    status=$(systemctl show -p ActiveState --value "$1")
+    if [ "$status" == "active" ]; then
+        sudo systemctl stop "$1"
+    fi
+}
+
+stop_services() {
+    stop_a_service_if_running "tedge-watchdog.service"
+    stop_a_service_if_running "tedge-mapper-collectd.service"
+    stop_a_service_if_running "c8y_log_plugin"
+    stop_a_service_if_running "c8y_configuration_plugin"
+    stop_a_service_if_running "apama"
```

Comment:
As far as I know, we don't need to stop apama.

Reply:
Added `apama` because it was mentioned in the ticket, removed.